darts/models/regression_ensemble_model.py

```diff
@@ -34,45 +40,42 @@ def __init__(self,
             Default: `sklearn.linear_model.LinearRegression(n_jobs=-1, fit_intercept=False)`
         """
         super().__init__(forecasting_models)
+        if regression_model is None:
+            regression_model = LinearRegression(n_jobs=-1, fit_intercept=False)
 
-        # wrap provided regression_model in a StandardRegressionModel (if not already the case)
-        if isinstance(regression_model, StandardRegressionModel):
-            # raise exception if train_n_points value is ambiguous
-            model_train_n_points = regression_model.train_n_points
-            raise_if(model_train_n_points is not None and regression_train_n_points != model_train_n_points,
-                     "Provided StandardRegressionModel.train_n_points parameter doesn't match specified"
-                     " regression_train_n_points parameter.",
-                     logger)
-
-            # if it was None, set regression_model.train_n_points to regression_train_n_points
-            regression_model.train_n_points = regression_train_n_points
-        else:
-            regression_model = StandardRegressionModel(regression_train_n_points, regression_model)
+        if not isinstance(regression_model, (LinearRegressionModel, RandomForest)):
+            regression_model = RegressionModel(lags_exog=[0], model=regression_model)
```

Comment:
Hm, I like when I know of what type an object is when handling it. Even if we only use the `fit()` and `predict()` methods we from now on know that it's an RegressionModel and what we can do with it.
But yes I don't think it would make any difference. I'll add some more tests here just to make sure everything works.

Reply:
I changed this whole section now to:
```python
if regression_model is None:
    regression_model = LinearRegressionModel(lags_exog=0, fit_intercept=False)

regression_model = RegressionModel(lags_exog=0, model=regression_model)
```
So we only rely on darts models in the ensemble regression (Of course one step down the line they depend on the corresponding scikit-learn models).

Reply:
On a side-note. The lags_exog argument of the `LinearRegressionModel` could be anything without making a difference as the `RegressionModel` has its own lags_exog parameter which will be used. So even though it wouldn't make much sense, the following code runs without problem:

```python
if regression_model is None:
    regression_model = LinearRegressionModel(lags_exog=12, fit_intercept=False)

regression_model = RegressionModel(lags_exog=0, model=regression_model)
```
I will mention this in the notebook, but do you think it's okey like that or should we check for this and raise an error?